examples/sagemaker-tensorflow/sagemaker-merlin-tensorflow.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>not sure what that means <code>Once you have tested your script that creates a Merlin ensemble graph, you can use it to train it on Sagemaker.</code> ? isnt training on SageMaker is a separate step then training on local? do you mean we will only do model deployment on SageMaker using the exported ensemble configs? if yes you need to rephrase <code>training</code> to <code>deployment</code>. it'd be useful if you can clarify. thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/692/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Thanks. <code>deployment</code> sounds better. Changed. </p>